subsys/bluetooth/services/fast_pair/fp_storage.c

```diff
@@ -111,7 +111,7 @@ static int fp_settings_set(const char *name, size_t len, settings_read_cb read_c
 			return -EINVAL;
 		}
 
-		memcpy(&account_key_list[index], data.account_key, sizeof(data.account_key));
+		memcpy(account_key_list[index], data.account_key, sizeof(data.account_key));
```

Comment:
Actually, you removed references here ;) Update commit message, please.

Reply:
fixed commit message